lib/model/response/me_response.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:survey_flutter_ic/model/response/converter/response_converter.dart';
+import 'package:survey_flutter_ic/model/user_model.dart';
+
+part 'me_response.g.dart';
+
+@JsonSerializable()
+class MeResponse {
```

Comment:
Would you mind changing the name of response to this, `MeResponse` is a little bit weird to me 😅 Also, the model was `UserModel` so I think the response should be `UserResponse`

```suggestion
class UserResponse {
```